synapse/cores/common.py

```diff
@@ -2301,6 +2301,7 @@ def _stormOperDset(self, query, oper):
 
     # some helpers to allow *all* queries to be processed via getTufosBy()
     def _tufosByEq(self, prop, valu, limit=None):
+        valu,_ = self.getPropNorm(prop,valu)
```

Comment:
nope, there's no way to norm them *before* the by handlers because we can't know what "norm" is. :D  ( inet:ipv4*inet:cidr=1.2.3.0/24 etc )